iree/vm/CMakeLists.txt

```diff
@@ -161,33 +161,24 @@ iree_cc_library(
     bytecode_module
   HDRS
     "bytecode_module.h"
+    "generated/bytecode_op_table.h"
   SRCS
     "bytecode_dispatch.c"
     "bytecode_dispatch_util.h"
     "bytecode_module.c"
     "bytecode_module_impl.h"
-    "bytecode_op_table.h"
```

Comment:
Hmm.. maybe keep the generated file in srcs?

https://docs.bazel.build/versions/master/be/c-cpp.html#cc_library.srcs
> The recommended style is for headers associated with a library to be listed in that library's hdrs attribute, and any remaining headers associated with this rule's sources to be listed in srcs

(the generated op table fits into the `srcs` category there)

Reply:
Done